internals/utils/src/index.ts

```diff
@@ -0,0 +1,393 @@
+export {default as link} from './output/link';
+export * from './output/list-item';
+export {default as listItem} from './output/list-item';
+export * from './output/logo';
+export {default as logo} from './output/logo';
+export * from './output/note';
+export {default as note} from './output/note';
+export * from './output/ok';
+export {default as ok} from './output/ok';
+export * from './output/param';
+export {default as param} from './output/param';
+export * from './output/progress';
+export * from './output/ready';
+export {default as ready} from './output/ready';
+export * from './output/routes';
+export {default as routes} from './output/routes';
+export * from './output/stamp';
+export {default as stamp} from './output/stamp';
+export * from './output/success';
+export {default as success} from './output/success';
+export * from './output/table';
+export {default as table} from './output/table';
+export * from './output/uid';
+export {default as uid} from './output/uid';
+export * from './output/wait';
+export {default as wait} from './output/wait';
+export * from './init/did-you-mean';
+export {default as didYouMean} from './init/did-you-mean';
+export * from './projects/add-domain-to-project';
+export * from './projects/create-project';
+export {default as createProject} from './projects/create-project';
+export * from './projects/find-framework';
+export * from './projects/find-projects-for-domain';
+export * from './projects/get-project-by-id-or-name';
+export {default as getProjectByIdOrName} from './projects/get-project-by-id-or-name';
+export * from './projects/link';
+export * from './projects/project-settings';
+export * from './projects/remove-domain-from-project';
+export * from './projects/remove-project';
+export {default as removeProject} from './projects/remove-project';
+export * from './projects/update-project';
+export {default as updateProject} from './projects/update-project';
+export * from './teams/create-team';
+export {default as createTeam} from './teams/create-team';
+export * from './teams/get-team-by-id';
+export {default as getTeamById} from './teams/get-team-by-id';
+export * from './teams/get-teams';
+export {default as getTeams} from './teams/get-teams';
+export * from './teams/invite-user-to-team';
+export {default as inviteUserToTeam} from './teams/invite-user-to-team';
+export * from './teams/patch-team';
+export {default as patchTeam} from './teams/patch-team';
+export * from './dev/builder';
+export * from './dev/errors';
+export * from './dev/headers';
+export * from './dev/is-url';
+export {default as isUrl} from './dev/is-url';
+export * from './dev/mime-type';
+export {default as mimeType} from './dev/mime-type';
+export * from './dev/parse-listen';
+export * from './dev/parse-query-string';
+export * from './dev/router';
+export * from './dev/server';
+export {default as server} from './dev/server';
+export * from './dev/types';
+export * from './parse-environment';
```

Comment:
Did you noticed any slowdown when using a barrel file like this? 

We discourage this in Next.js and there is even a feature to try to fix it 

https://nextjs.org/blog/next-13-1#import-resolution-for-smaller-bundles

Reply:
Wasn't aware that causes issues. And since the cli uses ncc to bundle everything together, the only impact would be at build time right? Build time seems similar to main branch locally for me.

The issue though is all of these files mix default and named exports. So now that all the code is moved, it is not easy for code in `packages/cli` to import things correctly. We'd have a mess of 

```js
import somethingDefault from '@vercel-internals/utils/fuzz';
import { somethingNamed } from '@vercel-internals/utils/buzz';
import somethingElseDefault, { somethingElseNamed } from '@vercel-internals/utils/foobar';
```

vs the much cleaner

```js
import {
  named1,
  named2,
  named3
} from '@vercel-internals/utils`;
```